core-bundle/src/Resources/contao/modules/ModuleRegistration.php

```diff
@@ -359,7 +359,8 @@ protected function compile()
 			$this->Template->$k = $v;
 
 			$key = $k . (($k == 'personal') ? 'Data' : 'Details');
-			$arrGroups[$GLOBALS['TL_LANG']['tl_member'][$key] ?? ''] = $v;
+			$legend = ($k == 'captcha') ? $GLOBALS['TL_LANG']['MSC']['securityQuestion'] : ($GLOBALS['TL_LANG']['tl_member'][$key] ?? '');
```

Comment:
I‘d prefer to add a translation for `tl_member.captchaDetails`, because as soon as we have other CAPTCHA variants, "security question" might no longer be accurate. WDYT?

Reply:
Fine by me 👍 . For now the new `tl_member.captchaDetails` translation should contain the same as `MSC.securityQuestion` though? Or something new entirely already?

Reply:
If you can think of something good, then yes.

Reply:
What do you think about: Ich bin kein Roboter / I am not a robot

Reply:
I went with @netzarbeiter 's suggestion.

Reply:
I‘m not sure about that. 🤔 

<img width="1088" alt="" src="https://github.com/contao/contao/assets/1192057/3f71ed75-505f-421f-9432-69cb891295fd">

Do we need a legend at all?

Reply:
I would be fine with no legend. In this case we can close this PR and only fix it in 5.3 via

```
$arrGroups[$GLOBALS['TL_LANG']['tl_member'][$key] ?? ''] = $v;
```

Or should we change the JS in any case? /cc @ausi

Reply:
What about _Security_ or _Verification_ as legend?

Reply:
I would then prefer _Verification_.